bee-common/bee-packable/tests/packable.rs

```diff
@@ -86,3 +86,23 @@ fn packable_vector() {
 fn packable_array() {
     assert_eq!(pack_checked([42u8; 1024]).len(), 1024 * size_of::<u8>());
 }
+
+macro_rules! packable_pack_prefix {
```

Comment:
Oh, this macro should be called `packable_vec_prefix` instead. :trollface:

Reply:
lol